server/db/add_hubs.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+original_dir=$(pwd)
+cd $(dirname "$0")
+
+if [[ $EUID > 0 ]]; then
+    echo "This script needs sudo privileges to run correctly."
+    cd $original_dir
+    exit 1
+elif [[ ! -n "$2" ]]; then
+    echo ""
+    echo "Usage: $0 path_to_.env_file path_to_hubs_csv"
+    echo "" 
+    echo "Example: $0 ./../.env ./brave_hubs.csv.example"
+    echo ""
+    echo "The hubs CSV file"
+    echo "MUST have the header 'system_id, system_name, heartbeat_alert_recipients'"
+    echo "MUST use Unix line endings (LF)"
+    echo "MUST end with a newline, or else the last hub will be silently ignored"
+    echo ""
+    cd $original_dir
+    exit 1
+else
+    while IFS="=" read -r name value; do
+        if [[ "$name" == "PG_USER" ]]; then
+            PG_USER="$value"
+        elif [[ "$name" == "PG_PASSWORD" ]]; then
+            PG_PASSWORD="$value"
+        elif [[ "$name" == "PG_HOST" ]]; then
+            PG_HOST="$value"
+        elif [[ "$name" == "PG_PORT" ]]; then
+            PG_PORT="$value"
+        fi
+    done < $1
+
+
+    while IFS=",", read -r system_id system_name heartbeat_alert_recipients; do
+        if [[ "$system_id" != "system_id" && "$system_id" != "" ]]; then
+            echo "Adding button"
```

Comment:
Do you mean `hub` instead of `button` here?

Reply:
this is captured in https://app.clickup.com/t/faxfbg